src/main/java/step2/domain/LottoNumber.java

```diff
@@ -0,0 +1,31 @@
+package step2.domain;
+
+public class LottoNumber {
+    private static final int MIN = 1;
+    private static final int MAX = 45;
+    private final int number;
+
+    private LottoNumber(int number) {
+        this.number = number;
+    }
+
+    public static LottoNumber of(int number) {
+        validate(number);
+        return new LottoNumber(number);
+    }
+
```

Comment:
정적 팩터리 메서드에 유효성 검증이 추가되었네요

주 생성자 안에 유효성 검증 로직이 있는 것이 자연스럽지 않을까요?
정적 팩터리 메서드가 생성자를 호출하고 있으니까요
```suggestion
    private LottoNumber(int number) {
        validate(number);
        this.number = number;
    }
    
    public static LottoNumber of(int number) {
        return new LottoNumber(number);
    }
```